### ISSUE TITLE
Input labels for admin edit pages

### ISSUE THREAD

**Issue by jaguillette** [omeka/Omeka#929]
Our university IT's accessibility group took a look at Omeka as provided to our community, and identified a few issues. One is that the text area inputs in the admin view don't always get descriptive labels. As they reported to us: 

"In the Admin pages input fields that are text areas are not given accessible names. This has the result of making them indistinguishable from one another to assistive technology. This [screen cast of a screen reader navigating the "Add an Item" page](https://share.getcloudapp.com/GGuRP51Q) shows how every text area is announced as "edit text". There is no way to identify to which section (e.g. Title, Subject, Description) the input corresponds. Note as well in the screen cast the "Use HTML unchecked checkbox" labels. In this case, although the input fields are given accessible names they are not descriptive and do not convey the context of the input."

I'm working on adding input labels for our own Omeka service, and will create a pull request referencing this issue when I have a solution to propose. I'm working off of the new-admin-theme branch so that the changes can be incorporated into the admin theme refresh if desirable.

**Comment by zerocrates** [comment]
This is great. And working off of the new-admin-theme branch is the right move, I think.

How are you currently planning to address this particular issue?

My thinking, when you described it eariler, was to use legends/fieldsets, or possibly aria-role=group and aria-labelledby, to provide the necessary context.

**Comment by jaguillette** [comment]
Adding a `for=` to the existing label would work for the first element, but then subsequent inputs would present a problem, so what seems likely right now is to add a visually hidden label to each textarea. I've done some work for #928 that adds a css class `visually-hidden`, and I'm likely to use the same approach here. The class styling looks like this:

    .visually-hidden {
    	border: 0 !important;
    	clip: rect(1px, 1px, 1px, 1px) !important;
    	-webkit-clip-path: inset(50%) !important;
    		clip-path: inset(50%) !important;
    	height: 1px !important;
    	margin: -1px !important;
    	overflow: hidden !important;
    	padding: 0 !important;
    	position: absolute !important;
    	width: 1px !important;
    	white-space: nowrap !important;
    }

**Comment by zerocrates** [comment]
We've used the group/label solution before, as described by https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA17.html, or the similar https://www.w3.org/WAI/WCAG21/Techniques/html/H71.html that uses legends.

Both don't have the downside of using `for` where the label only applies to one element, but also don't require repeating the group context over again for every element. We'll consider whatever you suggest, but my preference would be one of those two options.

**Comment by jaguillette** [comment]
Oh, those do look like more elegant implementations, thanks for sharing! I've gotten recommendations to use semantic HTML wherever possible, so I'll try the fieldset technique first, and might use the aria tags if that doesn't work out.
